### ISSUE TITLE
Enable block de-indent (shift+tab) in text editor? 

### ISSUE THREAD

**Issue by johnmarktaylor91** [jupyterlab/jupyterlab#6127]
I love the ability to highlight and de-indent a block of text in the Notebook editor with shift+tab. However, this feature doesn't seem to work in the text editor (i.e., if you do File->New->Text File)... would it be simple to add?

**Comment by Vonatzki** [comment]
This would be a big plus. I badly need this right now since I am trying to make jupyterlab as my main work env hosted on a remote VM.

Was surprised when I can't de-indent parts of my SQL script.

**Comment by v-iashin** [comment]
The same here

**Comment by BXSuhelSingh** [comment]
Yeah same here

**Comment by kuprel** [comment]
Block indent works, would be great to get de-indent as well

**Comment by gavieira** [comment]
I definitely agree

**Comment by gavieira** [comment]
Actually, I just found out that ***Ctrl-[*** works for block dedenting.

However, this shortcut did not work in chrome (probably due to some extension). Switching to another browser fixed the issue.

**Comment by ojsquire** [comment]
> Actually, I just found out that _**Ctrl-[**_ works for block dedenting.
> 
> However, this shortcut did not work for me in chrome (probably due to some extension). Switching to another browser fixed the issue.
> 
> Also, it seems relatively straightforward to [customize the shortcut](https://jupyterlab.readthedocs.io/en/stable/user/interface.html#keyboard-shortcuts) and replace it with shift+tab.

Thanks, this worked for me in Chrome (Version 79.0.3945.79 (Official Build) (64-bit)). I am on a Mac, so `Cmd-[` dedents, `Cmd-]` indents in a python script and a notebook.

**Comment by datdinhquoc** [comment]
Definitely a must, 'tab'-ed multiple lines, how to 'shift-tab' back?

**Comment by telamonian** [comment]
Strange that this works in notebook cells (which use codemirror) but not text editors (which use codemirror)...

Probably just needs a config fix somewhere

**Comment by telamonian** [comment]
Hah! I got it. It's the same problem that I fixed for the notebook and console waaay back in #5060: the 'shift tab' shortcut is also being used for the command to launch the interactive help tooltip. What's meant to happen is that if you just place the cursor at, say, a python keyword, 'shift tab' will open a tooltip with the relevant docstring; but if you have text selected and then hit 'shift tab' you get a de-dent. 

For whatever reason, the tooltip command is currently swallowing the 'shift tab' shortcut even when text is selected. I'll see if I can fix it
